### PR TITLE
Contract rename

### PR DESCRIPTION
**Pull request by leodsoft**

### CODE REVIEW

**Comment by leodsoft** [comment]
Guys - this is a draft contract rename, to check the impact and consider merging prior to contract audit.

I will be reviewing it further as well, as a bulk change it is possible there are some stray cases needing attention.

**Review by OliverNChalk — APPROVED** [review]
Looks good, build and test passes locally for me. Should we maintain `vexchange-diff` as a noise reduced diff for the auditors?

File: uniswap-v2-periphery/contracts/VexchangeV2Router01.sol [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-import './interfaces/IUniswapV2Router01.sol';
+import './libraries/VexchangeV2Library.sol';
+import './interfaces/IVexchangeV2Router01.sol';
 import './interfaces/IERC20.sol';
 import './interfaces/IWETH.sol';
 
-contract UniswapV2Router01 is IUniswapV2Router01 {
+contract VexchangeV2Router01 is IVexchangeV2Router01 {
     address public immutable override factory;
     address public immutable override WETH;
```
Comment (xenide) [inline uniswap-v2-periphery/contracts/VexchangeV2Router01.sol:13]: might need to change this to VVET

File: uniswap-v2-periphery/contracts/VexchangeV2Router01.sol [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-import './ext-v2-core/IUniswapV2Factory.sol';
+import './ext-v2-core/IVexchangeV2Factory.sol';
 import './ext-lib/TransferHelper.sol';
 
-import './libraries/UniswapV2Library.sol';
-import './interfaces/IUniswapV2Router01.sol';
+import './libraries/VexchangeV2Library.sol';
+import './interfaces/IVexchangeV2Router01.sol';
 import './interfaces/IERC20.sol';
 import './interfaces/IWETH.sol';
```
Comment (xenide) [inline uniswap-v2-periphery/contracts/VexchangeV2Router01.sol:9]: same thing

**Review by xenide — COMMENTED** [review]
something came to my mind while I was writing the deployment scripts. We need to change all instances of WETH to VVET. Perhaps this can be done in the phase after this one

**Comment by leodsoft** [comment]
Thx @xenide - yeah i had created a separate trello task to handle the WVET updates (& was leaving until we have confirmed whether we will definitely add our own WVET contract or can rely on the previous effort).

**Review by xenide — APPROVED** [review]

**Pull request merged**